src/directory/directory.js

```diff
@@ -288,6 +288,11 @@ const directory = {
             route: '/lib/auth/signin',
             filters: ['android', 'flutter', 'ios']
           },
+          {
```

Comment:
can you please use the existing Switching authentication flows directory entry? Rather than creating another one just for iOS?

Reply:
Similar is the case here.. If I use the existing one, Switch Auth would be the first option whereas we want it to come after sign in..